books.js

```diff
@@ -0,0 +1,118 @@
+const bookList = [];
+
+class Books {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+const section = document.querySelector('.bookings');
+let count = 0;
+
+function addBook(Title, Author) {
+  const booky = new Books(Title, Author);
+  bookList.push(booky);
+
+  const container = document.createElement('div');
+  const names = document.createElement('h3');
+  const authors = document.createElement('h4');
+  const button = document.createElement('button');
+  const line = document.createElement('span');
+
+  section.append(container);
+  container.append(names);
+  container.append(authors);
+  container.append(button);
+  container.append(line);
+
+  const counting = `count${count}`;
+  const buttoning = `button${count}`;
+
+  container.setAttribute('class', 'booksection');
+  container.setAttribute('id', counting);
+  names.innerText = booky.title;
+  authors.innerText = booky.author;
+  button.setAttribute('class', 'removebutton');
+  button.setAttribute('id', buttoning);
+  button.innerHTML = 'Remove';
+  line.setAttribute('class', 'line');
+  count += 1;
+}
+
+function removeList(name, author) {
+  for (let x = 0; x < bookList.length; x += 1) {
+    if ((bookList[x].title === name) && (bookList[x].author === author)) {
+      bookList.splice(x, 1);
+    }
+  }
+}
+
+function removeBook(args) {
```

Comment:
**Kindly fix this issue displayed in console** 
- When i tried to remove an item and this error displayed:
![Screen Shot 2021-11-23 at 7 34 44](https://user-images.githubusercontent.com/22258274/143034262-1784cca0-4300-4c40-86ba-e69a3bdf99b0.png)